packages/react-reconciler/src/__tests__/ReactTransitionTracing-test.js

```diff
@@ -1362,4 +1365,365 @@ describe('ReactInteractionTracing', () => {
       ]);
     });
   });
+
+  // @gate enableTransitionTracing
+  it('offscreen trees should not stop transition from completing', async () => {
```

Comment:
It looks like this test would pass even before this PR? Could you open a PR that includes just test to confirm?

Then since the rest is related to "deletions" I would group it with that PR.